Tribler/dependencies.py

```diff
@@ -0,0 +1,72 @@
+"""
+This file lists the python dependencies for Tribler.
+
+Note that this file should not depend on any external modules itself other than builtin ones.
+"""
+from __future__ import absolute_import, print_function
+
+import importlib
+import sys
+
+dependencies = [
+    {'module': 'PyQt5', 'pip_install': 'pip3 install PyQt5', 'optional': False, 'scope': 'gui'},
+    {'module': 'twisted', 'pip_install': 'pip3 install Twisted', 'optional': False, 'scope': 'core'},
+    {'module': 'libtorrent', 'pip_install': 'apt install python-libtorrent', 'optional': False, 'scope': 'core'},
+    {'module': 'cryptography', 'pip_install': 'pip3 install cryptograpy>=2.3', 'optional': False, 'scope': 'core'},
+    {'module': 'libnacl', 'pip_install': 'pip3 install libnacl', 'optional': False, 'scope': 'core'},
+    {'module': 'pony', 'pip_install': 'pip3 install pony', 'optional': False, 'scope': 'core'},
+    {'module': 'lz4', 'pip_install': 'pip3 install lz4', 'optional': False, 'scope': 'core'},
+    {'module': 'psutil', 'pip_install': 'pip3 install psutil', 'optional': False, 'scope': 'both'},
+    {'module': 'networkx', 'pip_install': 'pip3 install networkx', 'optional': False, 'scope': 'both'},
+    {'module': 'pyqtgraph', 'pip_install': 'pip3 install pyqtgraph', 'optional': False, 'scope': 'gui'},
+    {'module': 'matplotlib', 'pip_install': 'pip3 install matplotlib', 'optional': False, 'scope': 'gui'},
+    {'module': 'chardet', 'pip_install': 'pip3 install chardet', 'optional': False, 'scope': 'core'},
+    {'module': 'cherrypy', 'pip_install': 'pip3 install cherrypy', 'optional': False, 'scope': 'core'},
+    {'module': 'configobj', 'pip_install': 'pip3 install configobj', 'optional': False, 'scope': 'both'},
+    {'module': 'netifaces', 'pip_install': 'pip install netifaces', 'optional': False, 'scope': 'core'},
+    {'module': 'six', 'pip_install': 'pip install six', 'optional': False, 'scope': 'both'},
+    {'module': 'bitcoinlib', 'pip_install': 'pip install bitcoinlib', 'optional': True, 'scope': 'core'},
+]
+
+
+def show_system_popup(title, text):
```

Comment:
Could you make this private? Nobody should ever use this outside of this file.

Reply:
Good point. Done